src/image/imageengine.js

```diff
@@ -52,8 +54,18 @@ export default class ImageEngine extends Plugin {
 		createImageAttributeConverter( [ editing.modelToView, data.modelToView ], 'src' );
 		createImageAttributeConverter( [ editing.modelToView, data.modelToView ], 'alt' );
 
+		// Build converter for view img element to model image element.
+		buildViewConverter().for( data.viewToModel )
+			.from( { name: 'img', attribute: { src: /.+/ } } )
```

Comment:
`src` attribute is required to correctly convert view `<img>` to model `<image>` (which requires `src` attribute). If I understand the code correctly :), this was also implemented in "old" converter (before changes introduced in this PR): https://github.com/ckeditor/ckeditor5-image/pull/97/files#diff-ac5209f1bc4dd7be6f3619d8632013c9L43 here and below.